docs/ABI.rst

```diff
@@ -1027,6 +1031,12 @@ Types
   METATYPE-REPR ::= 'T'                      // Thick metatype representation
   METATYPE-REPR ::= 'o'                      // ObjC metatype representation
 
+  SIL-BOX-FIELD-MUTABILITY ::= 'm'           // mutable field
+  SIL-BOX-FIELD-MUTABILITY ::= 'i'           // immutable field
+                                             // The number of mutability
+                                             // specifiers matches the length
+                                             // of the preceding type-list.
+
```

Comment:
I think it's better to explicitly terminate the list of 'm'/'i', e.g. with a trailing _ (and not just derive the length from the preceding type list). It's also the case for all other operators that it's easy to figure out where they end.
And I think the additional '_' does not hurt